src/main/java/org/mitre/synthea/export/FhirR4.java

```diff
@@ -191,6 +194,56 @@ public class FhirR4 {
   private static final Table<String, String, String> US_CORE_MAPPING =
       loadMapping("us_core_mapping.csv");
 
+  private static final HashSet<String> includedResources = new HashSet<>();
+  private static final HashSet<String> excludedResources = new HashSet<>();
+
+  static {
+    reloadIncludeExclude();
+  }
+
+  static void reloadIncludeExclude() {
+    includedResources.clear();
+    excludedResources.clear();
+    String includedResourcesStr = Config.get("exporter.fhir.included_resources", "").trim();
+    String excludedResourcesStr = Config.get("exporter.fhir.excluded_resources", "").trim();
+
+    List<String> includedResourcesList = Collections.emptyList();
+    List<String> excludedResourcesList = Collections.emptyList();
+
+    if (!includedResourcesStr.isEmpty() && !excludedResourcesStr.isEmpty()) {
+      System.err.println(
+          "FHIR exporter: Included and Excluded resource settings are both set -- ignoring both");
+    } else if (!includedResourcesStr.isEmpty()) {
+      includedResourcesList = propStringToList(includedResourcesStr);
+    } else if (!excludedResourcesStr.isEmpty()) {
+      excludedResourcesList = propStringToList(excludedResourcesStr);
+    }
+
+    includedResources.addAll(includedResourcesList);
+    excludedResources.addAll(excludedResourcesList);
+  }
+
+  static boolean shouldExport(String resourceType) {
+    return (includedResources.isEmpty() || includedResources.contains(resourceType))
+            && !excludedResources.contains(resourceType);
+  }
+
+  /**
+   * Helper function to convert a string of resource type names
+   *  from synthea.properties into a list of FHIR ResourceTypes.
+   * @param propString String directly from Config, ex "Patient,Condition , Procedure"
+   * @return normalized list of filenames as strings
+   */
+  private static List<String> propStringToList(String propString) {
+    List<String> files = Arrays.asList(propString.split(","));
+    // normalize filenames by trimming
+    files = files.stream().map(f ->  f.trim()).collect(Collectors.toList());
+
+    // TODO: convert these into FHIR resource enums
```

Comment:
Instead of using an enum you could use the class. E.g. `Class clazz = Class.forName("org.hl7.fhir.r4.model." + className)`.
This function could return a `List<Class>` instead of `List<String>`
The `Class.forName` method can throw a ClassNotFound exception. You could use that to catch spelling errors in the list of resources to include/exclude.